meshstack-hub/.prettierrc.json

```diff
@@ -0,0 +1,9 @@
+{
+  "printWidth": 120,       
```

Comment:
d: Please make sure that the configuration matches the eslint rules.

Reply:
done